pkg/apis/serving/v1alpha1/kfservice_framework_tensorrt.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	TensorRTISImageName             = "nvcr.io/nvidia/tensorrtserver"
+	DefaultTensorRTISRuntimeVersion = "19.05-py3"
+	TensorRTISEntrypointCommand     = "trtserver"
+	InvalidModelURIError            = "Model URI must be prefixed by gs:// (only Google Cloud Storage paths are supported)"
+	TensorRTISGRPCPort              = int32(9000)
+	TensorRTISRestPort              = int32(8080)
+)
+
+func (t *TensorRTSpec) CreateModelServingContainer(modelName string, config *FrameworksConfig) *v1.Container {
+	imageName := TensorRTISImageName
+	if config.TensorRT.ContainerImage != "" {
+		imageName = config.TensorRT.ContainerImage
+	}
+
+	// based on example at: https://github.com/NVIDIA/tensorrt-laboratory/blob/master/examples/Deployment/Kubernetes/basic-trtis-deployment/deploy.yml
+	return &v1.Container{
+		Image:     imageName + ":" + t.RuntimeVersion,
+		Command:   []string{TensorRTISEntrypointCommand},
+		Resources: t.Resources,
+		Args: []string{
+			"--model-store=" + t.ModelURI,
+			"--allow-poll-model-repository=false",
+			"--allow-grpc=true",
+			"--allow-http=true",
+			"--grpc-port=" + fmt.Sprint(TensorRTISGRPCPort),
+			"--rest-port=" + fmt.Sprint(TensorRTISRestPort),
```

Comment:
here should be `--http-port=`, I got this error while launching the container.
```
(katie) C02R2BFYG8WP:kfserving dsun20$ kubectl logs simple-string-default-rzf89-deployment-758949ffdb-7jfws -c user-container
trtserver: unrecognized option '--rest-port=8080'
E0614 15:28:22.734684 1 main.cc:392] Usage:
--help	Print usage
--log-verbose	Enable/disable verbose-level logging
--log-info	Enable/disable info-level logging
--log-warning	Enable/disable warning-level logging
--log-error	Enable/disable error-level logging
--id	Identifier for this server
--model-store	Path to model repository directory
--exit-on-error	Exit the inference server if an error occurs during initialization.
--strict-model-config	If true model configuration files must be provided and all required configuration settings must be specified. If false the model configuration may be absent or only partially specified and the server will attempt to derive the missing required configuration.
--strict-readiness	If true /api/health/ready endpoint indicates ready if the server is responsive and all models are available. If false /api/health/ready endpoint indicates ready if server is responsive even if some/all models are unavailable.
--allow-profiling	Allow server profiling.
--allow-grpc	Allow the server to listen for GRPC requests.
--allow-http	Allow the server to listen for HTTP requests.
--allow-metrics	Allow the server to provide prometheus metrics.
--allow-gpu-metrics	Allow the server to provide GPU metrics. Ignored unless --allow-metrics is true.
--grpc-port	The port for the server to listen on for GRPC requests.
--http-port	The port for the server to listen on for HTTP requests.
```

Reply:
fixed